mapeditor-appdata.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/org.mapeditor.Tiled.appdata.xml b/org.mapeditor.Tiled.appdata.xml
+index 5c7dd243..7f25e1c6 100644
+--- a/org.mapeditor.Tiled.appdata.xml
++++ b/org.mapeditor.Tiled.appdata.xml
+@@ -1,6 +1,6 @@
+ <?xml version="1.0" encoding="UTF-8"?>
+ <component type="desktop">
+- <id>org.mapeditor.Tiled.desktop</id>
++ <id>org.mapeditor.Tiled</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0+ and BSD-2-Clause</project_license>
+  <name>Tiled</name>
+@@ -161,7 +161,6 @@
+  <url type="help">https://doc.mapeditor.org/</url>
+  <url type="donation">https://www.mapeditor.org/donate</url>
+  <url type="translate">https://hosted.weblate.org/projects/tiled/translations/</url>
+- <url type="vcs-browser">https://github.com/mapeditor/tiled</url>
```

Comment:
Why remove this URL?

Reply:
it cause the validation to fail. Not sure what changed with appstream again, but with this line it's fatal the build fails.

Reply:
more like appstream-compose fail, which is much earlier.

Reply:
I have removed this. It's caused by a discrepency between upstream flatpak-builder (that fails) and the version used on flathub. We need to get to the bottom of it, but not this time.

Reply:
Ok, thanks!